src/App.css

```diff
@@ -514,26 +287,36 @@ button {
   margin-right: 0;
 }
 
-.calendarVariation {
-  width: 20px;
-  height: 20px;
-  font-size: 8pt;
-  margin-right: 0;
-}
-
 .filterButton {
   background-color: var(--purple1);
   width: 25px;
   height: 25px;
 }
 
-.crossButton {
+.xButtonContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 250px; 
+  height: 0px; 
+  justify-content: flex-end;
+}
+.xButton {
   background-color: var(--white);
   /* placeholder */
   width: 10px;
   height: 10px;
 }
 
+.sendSaveContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 250px;
+  height: 0px;
+  gap: 5px;
+  justify-content: flex-end;
+}
 .sendSaveButtons {
```

Comment:
I recommend trying [css combinators](https://css-tricks.com/css-selectors/#combinators) to reduce the number of classnames you have to use in your code.
In this example, instead of creating a new CSS class `.sendSaveButtons`, you can use the selector `.sendSaveContainer button` which selects all the button elements inside a sendSaveContainer. Then in your definition of the `SaveButton` and `SendButton` components, you don't need to add a className to the `<button>` tag.